src/cli/Peers.ts

```diff
@@ -0,0 +1,387 @@
+import fs from 'fs';
+import commander from 'commander';
+import { PolykeyAgent, PeerInfo } from '../lib/Polykey';
+import {} from '../../tmp/hole-punch-test/publicserver';
+import { actionRunner, pkLogger, PKMessageType, determineNodePath } from '.';
+
+function makeAddPeerCommand() {
+  return new commander.Command('add')
+    .description('add a new peer to the store')
+    .option('-b, --base64 <base64>', 'decode the peer info from a base64 string')
+    .option('-p, --public-key <publicKey>', 'path to the file which contains the public key')
+    .option('-a, --connected-address <connectedAddress>', 'public address on which the node can be contacted')
+    .option('-r, --relay-key <relayKey>', 'path to the file which contains the public key of the relay peer')
+    .option('--node-path <nodePath>', 'node path')
+    .option('-v, --verbose', 'increase verbosity level by one')
+    .action(
+      actionRunner(async (options) => {
+        const client = PolykeyAgent.connectToAgent();
+        const status = await client.getAgentStatus();
+        if (status != 'online') {
+          throw Error(`agent status is: ${status}`);
```

Comment:
oh okay, I just figured since they are source files and not really binary files that bin isn't really accurate. 

Yeah I guess the file name shouldn't really be uppercase since it's not exporting a class but I think the cli and the lib are different things, it doesn't make sense to me to have the `peer` command inside the library as it doesn't compile any command like that.

Reply:
I've renamed them to be all lower case file names :+1: